code/src/components/AddTask.js

```diff
@@ -0,0 +1,132 @@
+import React, { useState } from 'react';
+import { useDispatch } from 'react-redux';
+import Select from 'react-select';
+import styled from 'styled-components';
+import Datepicker from 'react-date-picker';
+import 'index.css';
+
+import { tasks } from 'reducers/tasks';
+
+export const AddTask = () => {
+  const [text, addTask] = useState('');
```

Comment:
I would maybe name this `[text, setText]` or `[task, setTask]` to prevent confusion with the action